lib/rubocop/cop/style/ip_addresses.rb

```diff
@@ -52,7 +52,7 @@ def allowed_addresses
           Array(allowed_addresses).map(&:downcase)
         end
 
-        def could_be_ip?(str)
+        def potential_ip?(str)
```

Comment:
While checking for `can/could` prefixes I came across this which I think *is* a good candidate for removing `could` without adding complexity.

Reply:
Agreed. I often use names like `maybe_ip?` in such cases. (habit from a lot of Lisp programming)

Reply:
The thing with `maybe` is that it's just hiding another auxiliary verb (may) 😆

Reply:
Yeah, yeah - this was just a random comment. 😀